.changeset/fast-jeans-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@urql/core": minor
+---
+
+implement new defer/stream spec
```

Comment:
```suggestion
---
"@urql/core": patch
---

Implement new `@defer` / `@stream` transport protocol spec changes.
```

Reply:
I'd love to designate this as a patch, if you don't mind, since it's not stable yet and the minor version update doesn't do much as long as there's no corresponding minor change in the GraphQL package 😅